src/test/java/co/rsk/federate/signing/hsm/client/HSMClientProtocolTest.java

```diff
@@ -175,7 +175,7 @@ public void sendInvalidVersion() throws JsonRpcException  {
         ObjectNode expectedRequest = new ObjectMapper().createObjectNode();
         expectedRequest.put("command", command);
         expectedRequest.put("version", version);
-        ObjectNode sendResponse = buildResponse(-666);
+        ObjectNode sendResponse = buildResponse(-904);
```

Comment:
Why this change?

Reply:
-666 Is No Longer The Code For That Particular Error, It Wasn't Updated All This While